analytics.py

```diff
@@ -0,0 +1,129 @@
+import math
+from assignment_05 import utils
+
+def check_significant(lower, upper, observed_avg):
+    significance = False
+    if(upper - lower <= observed_avg):
+        significance = True
+    return significance 
+
+def compute_critical(listOfAvgNNDistances):
+    criticalPoints = []
+    criticalPoints.append(min(listOfAvgNNDistances))
+    criticalPoints.append(max(listOfAvgNNDistances))
+    return criticalPoints;

```

Comment:
The semicolon works here because Python thinks you are going to add another line (on the same line, a la `x = 1; y = 2`.  I am guessing you are coming from a `;` language?